version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "2.0.0"
+	Version = "2.0.2"
```

Comment:
why skip `2.0.1`?

Reply:
Chart with version 2.0.1 already exists in S3 (check test03 cluster -> wizard -> the link points tohttps://setup.test.containers.carbonblack.io/operator-v2.0.1 ). 

I am not sure what is the production version of that link to validate if it exists there as well. 
It utilizes operator version 2.0.0. Not sure why 2.0.1 is not modified here - but to be on the safe side, I think it makes sense to utilize 2.0.1.